### ISSUE TITLE
conv type为2d时求theta是不是有点问题

### ISSUE THREAD

**Issue by WuChannn** [bobo0810/DynamicReLU#1]
原代码中：
if self.conv_type == '2d':
     theta = torch.mean(theta, axis=-1)
但是这样的话theta为三维的，是不是应该在求mean之前reshape，如：
if self.conv_type == '2d':
     theta = torch.reshape(theta, (:,:,-1))
     theta = torch.mean(theta, axis=-1)

**Comment by WuChannn** [comment]
@bobo0810 期待与您探讨一下

**Comment by bobo0810** [comment]
> @bobo0810 期待与您探讨一下

你好，非常抱歉，我在训练过程中没有遇到这个问题。也许你可以去[官方库](https://github.com/Islanna/DynamicReLU)探讨一下？
